src/lib/registration/registration_ui.rb

```diff
@@ -55,7 +55,7 @@ def initialize(registration)
     def register_system_and_base_product
       product_service = nil
       # TRANSLATORS: Popup error message prefix
-      error_options = { message_prefix: _("Registration failed.") }
+      error_options = { message_prefix: _("Registration failed.") + "\n\n" }
```

Comment:
Added new lines to the message prefix:

![Screenshot_testing_2019-03-13_09:06:14](https://user-images.githubusercontent.com/7056681/54266600-9da47d00-456f-11e9-9347-9401bd5c4b5b.png)